samples/runtime/reversible-simulator/Simulator.cs

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using System.Collections.Generic;
+using Microsoft.Quantum.Simulation.QuantumProcessor;
+using Microsoft.Quantum.Simulation.Common;
+using Microsoft.Quantum.Simulation.Core;
+
+namespace Microsoft.Quantum.Samples {
```

Comment:
Similarly here, while braces-in-line is what we've used in Q#, C# style tends to put them on the next line.
```suggestion
namespace Microsoft.Quantum.Samples
{
```